flow360/component/geometry.py

```diff
@@ -156,6 +100,8 @@ def _validate_geometry(self):
             raise Flow360ValueError(
                 "name field is required if more than one geometry files are provided."
             )
+        if self.length_unit not in LengthUnitType.__args__:
+            raise Flow360ValueError(f"specified length_unit : {self.length_unit} is invalid.")
```

Comment:
mention also what "unit" is expected.
By the way, what if user want to use "2m" as a mesh unit?

Reply:
Done.

Exactly my confusion.

There is no place on webUI to select the magnitude for mesh unit. Only selecting the unit.

Though Runda confirmed that 99% of the time user expect 1 for the magnitude.